koin-projects/koin-core/src/main/kotlin/org/koin/core/KoinComponent.kt

```diff
@@ -43,6 +43,19 @@ inline fun <reified T> KoinComponent.get(
 ): T =
     getKoin().get(qualifier, parameters)
 
+/**
+ * Get a Koin instance if available
+ * @param qualifier
+ * @param parameters
+ *
+ * @return instance of type T or null
+ */
+inline fun <reified T> KoinComponent.getOrNull(
```

Comment:
Need to add @KoinApiExtension

Reply:
Since it was developed based on the master branch before, I didn't notice this part. Now it has been corrected, thanks.